### DISCUSSION TITLE
This user is blocked

### DISCUSSION THREAD

**Discussion by myj47** [vercel/community#4401] [Account Issue] [answered]
### Question

Hi, I hope all of you are doing well. I am a student I have just started learning web development. I signed up using same account but I was unable . I received a message this user is blocked. I tried on a second email. I verified it but I am also unable to sign up on that's email too. Please help me in this regard so that I continue my development journey.

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble creating a new account. I know it's frustrating to get stuck like that. 🙁 

We can't safely discuss account information in public because of security concerns. To open a ticket, please use the appropriate [direct channel](https://vercel.com/guides/how-to-get-vercel-support#direct-channels) found in this guide: [What is the best way to get support from Vercel?](https://vercel.com/guides/how-to-get-vercel-support#direct-channels)

**Comment by niruallen** [comment]
Having the same issue 🤦‍♂️

**Reply by niruallen** [reply]
I have been waiting for your reply for more than 7 days now!!!
